From 5e95c0950a52153da70b8cd48287f78103622967 Mon Sep 17 00:00:00 2001
From: Benjamin Berg <bberg@redhat.com>
Date: Thu, 1 Oct 2020 11:44:46 +0200
Subject: [PATCH] data: Restart systemd service on failure
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

It makes sense to try restarting thermald should it ever fail. Add the
appropriate configuration to the systemd unit.
---
 data/thermald.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/thermald.service.in b/data/thermald.service.in
index 49de893..b2ecafe 100644
--- a/data/thermald.service.in
+++ b/data/thermald.service.in
@@ -7,6 +7,7 @@ Type=dbus
 SuccessExitStatus=2
 BusName=org.freedesktop.thermald
 ExecStart=@sbindir@/thermald --systemd --dbus-enable --adaptive
+Restart=on-failure
 
 [Install]
 WantedBy=multi-user.target
-- 
2.27.0

